src/Compiler.php

```diff
@@ -47,6 +47,7 @@ class Compiler
         'Slack',
         'Discord',
         'Telegram',
+        'Teams',
```

Comment:
Maybe we should rename this (and all other references) to `MicrosoftTeams`? I realize "Teams" is the app name but it's so generic that it could be confusing.

Reply:
I agree with you, I'll working on this.

Reply:
@MIsmail80 I've converted the PR to draft until you've made these changes. Feel free to mark it as ready again when you've done the changes.